src/pages/DatasetCatalog.js

```diff
@@ -506,17 +514,16 @@ class DatasetCatalog extends Component {
                             get(find(dacs, dac => { return dac.id === dataSet.dacId; }), 'name', '')
                           ]),
 
-                          td({ className: 'table-items cell-size ' + (!dataSet.active ? 'dataset-disabled' : '') }, [
-                            a({
-                              id: trIndex + '_linkdbGap',
-                              name: 'link_dbGap',
-                              href: get(find(dataSet.properties, p => { return p.propertyName === 'dbGAP'; }), 'propertyValue', ''),
-                              target: '_blank',
-                              className: (get(find(dataSet.properties, p => { return p.propertyName === 'dbGAP'; }), 'propertyValue', '').length > 0 ?
-                                'enabled' :
-                                'disabled')
-                            }, ['Link'])
-                          ]),
+                            td({ className: 'table-items cell-size ' + (!dataSet.active ? 'dataset-disabled' : '') },
+                              [ dataSet.dbGapLink !== '' ?
+                                  a({
+                                  id: trIndex + '_linkdbGap',
+                                  name: 'link_dbGap',
+                                  href: dataSet.dbGapLink,
+                                  target: '_blank',
+                                  className: 'enabled'
+                                }, ['Link']) : ''
```

Comment:
Double check with Jonathan about whether we need to display a `--` instead of an empty string in this column when no link is available.